src/assets/css/components/button.css

```diff
@@ -0,0 +1,19 @@
+.btn {
+  color: var(--primary-color);
+  font-family: var(--primary-font);
+  font-size: 16px;
+  font-weight: var(--fw-semibold);
+  padding: 10px 25px 10px 25px;
```

Comment:
Use short-hand please

Reply:
sorry, i fixed it to padding: 10px 25px;